tabpfn_client/prompt_agent.py

```diff
@@ -115,3 +126,24 @@ def prompt_confirm_password_for_user_account_deletion(cls) -> str:
     @classmethod
     def prompt_account_deleted(cls):
         print(cls.indent("Your account has been deleted."))
+
+    @classmethod
+    def _choice_with_retries(cls, prompt: str, choices: list, max_attempts: int = 3) -> str:
+        """
+        Prompt text and give user predefined number of attempts to select one of the possible choices. If valid choice
+        is selected, return choice in lowercase, otherwise raise RuntimeError.
+        """
```

Comment:
add assert to only have lower cased choices
```suggestion
        """
        assert all(c.lower() == c for c in choices), "Choices need to be lower case."
```